lib/migration.js

```diff
@@ -80,8 +84,33 @@ const fetchMigration = async (config) => {
     const filename = path.join(directory, `${timestamp++}-create-${entry.sys.id}-migration.${module ? 'cjs' : 'js'}`);
 
     const content = await generateMigrationScript(client, [entry]);
+    // Fetch migration script generated by contentful
+    let modifiedContent = content.toString();
+    // Modify migration script to use our helpers
+    const testDefaultValueRegex = /(defaultValue\({\s*)[^\:]+([^\)]+)/g;
+    if (testDefaultValueRegex.test(modifiedContent)) {
+      modifiedContent = content
+        .toString()
+        // Add call to utils.getDefaultLocale() to the top
+        .replace(
+          'module.exports = function (migration) {',`
+          ${migrationHeader}
+          const defaultLocale = helpers.locale.getDefaultLocale();
+        `)
+        // Replace the default locale with defaultLocale.code so that the migration
+        // still works as expected when the locale is changed in contentful
+        .replace(testDefaultValueRegex, '$1[defaultLocale.code]$2')
+        // Add a closing parentheses as we wrap the migration function.
+        .replace(/};\s*$/g, '});');
+    } else {
+      // If we don't have a default value we just wrap the migration function with our withHelpers wrapper
+      modifiedContent = content
+        .toString()
+        .replace('module.exports = function (migration) {', migrationHeader)
```

Comment:
In diesen Beiden zeilen sind die search strings auch gleich wie oben.
Eine eigene Funktion wäre da vielleicht doch hilfreich gewesen.